ranger/default.go

```diff
@@ -27,10 +28,12 @@ const (
 	baseURLEnvVar = "BASE_URL"
 
 	// App metadata
-	appDescEnvVar   = "APP_DESCRIPTION"
-	defaultAppDesc  = "A trails app"
-	appTitleEnvVar  = "APP_TITLE"
-	defaultAppTitle = "trails"
+	appDescEnvVar    = "APP_DESCRIPTION"
+	defaultAppDesc   = "A trails app"
```

Comment:
These "app metadata" env vars are good examples of required ones that do not have a useful default value. There's a guard that needs to be put in place to validate the values pulled from the environment are useful. Undecided where that should live at the moment.

You can contrast this with bad values for DB connection env vars just means the connection fails and the user gets the appropriate error message.